src/modules/filelist/helpers.ts

```diff
@@ -20,22 +24,22 @@ export const getFileNameAndExtension = (
   filename: string
   extension?: string
 } => {
-  const { filename, extension } = splitFilename(file)
-
   if (file._id === TRASH_DIR_ID) {
     return {
       title: t('FileName.trash'),
       filename: t('FileName.trash')
     }
   }
 
-  if (file._id === 'io.cozy.files.shared-drives-dir') {
+  if (file._id === SHARED_DRIVES_DIR_ID || file._id === ROOT_DIR_ID) {
     return {
       title: t('FileName.sharedDrive'),
       filename: t('FileName.sharedDrive')
     }
   }
```

Comment:
Why if `file._id === ROOT_DIR_ID` we return this ?

Reply:
yes good question. Because in sharing view, with a hard refresh, we fetch root folder too so we have to do the same exception as for shared_drives_dir_id

Reply:
Ok its clearly not self explanatory